### PR TITLE
Feature aur#142 cleanup sis repository

### PR DESCRIPTION
**Pull request by Xarpagon**

### CODE REVIEW

File: test/integration/Makefile [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+.SILENT :
+	${TESTS}
+
+.PHONY :
+	${TESTS}
+	check
+
+##############################INTEGRATION TESTS################################
+
+run_rtems:
```
Comment (Lurkerpas) [inline test/integration/Makefile:24]: Please correct me if I am wrong, but the actual test is executed when the RESULT variable is assigned, right? I would prefer it to be executed within the rule

Also, if we are to include more integration tests in the future, either the file should be prepared to handle more, or we should prepare to have more makefiles.

In other tests done in makefile, we simply propagate the failure of a grep. Or, we can do exit 1.

File: test/integration/Makefile [hunk 2]
```diff
@@ -0,0 +1,30 @@
+SIS_NAME = sis
```
Comment (Lurkerpas) [inline test/integration/Makefile:1]: Test not ivoked on CI

File: test/integration/Makefile (hunk 1 above)
Comment (Xarpagon) [reply test/integration/Makefile]: Ack

File: test/integration/Makefile (hunk 2 above)
Comment (Xarpagon) [reply test/integration/Makefile]: Ack

**Review by Lurkerpas — APPROVED** [review]

**Pull request merged**